src/graph/graph.c

```diff
@@ -41,6 +43,22 @@ void _edge_accum(void *_z, const void *_x, const void *_y) {
     }
 }
 
+bool _select_op_free_edge(GrB_Index i, GrB_Index j, GrB_Index nrows, GrB_Index ncols, const void *x, const Graph *g) {
```

Comment:
Use a `const void *` instead of a `const Graph*` to fix a compiler warning for me (casting the pointer in-function is fine).

Reply:
👍